App crashe

Just open the app

> FATAL EXCEPTION: main
Process: com.forrestguice.suntimes.intervalmidpoints, PID: 16659
java.lang.RuntimeException: Unable to start activity ComponentInfo{com.forrestguice.suntimes.intervalmidpoints/com.forrestguice.suntimes.intervalmidpoints.MainActivity}: android.view.InflateException: Binary XML file line #1 in com.forrestguice.suntimes.intervalmidpoints:layout/design_layout_snackbar_include: Binary XML file line #1 in com.forrestguice.suntimes.intervalmidpoints:layout/design_layout_snackbar_include: Error inflating class x
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3983)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:4136)
	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:103)
	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:149)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:99)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2581)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loopOnce(Looper.java:224)
	at android.os.Looper.loop(Looper.java:318)
	at android.app.ActivityThread.main(ActivityThread.java:8727)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:561)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1013)
Caused by: android.view.InflateException: Binary XML file line #1 in com.forrestguice.suntimes.intervalmidpoints:layout/design_layout_snackbar_include: Binary XML file line #1 in com.forrestguice.suntimes.intervalmidpoints:layout/design_layout_snackbar_include: Error inflating class x
Caused by: android.view.InflateException: Binary XML file line #1 in com.forrestguice.suntimes.intervalmidpoints:layout/design_layout_snackbar_include: Error inflating class x
Caused by: java.lang.ClassNotFoundException: android.view.x
	at java.lang.Class.classForName(Native Method)
	at java.lang.Class.forName(Class.java:607)
	at android.view.LayoutInflater.createView(LayoutInflater.java:840)
	at android.view.LayoutInflater.createView(LayoutInflater.java:801)
	at android.view.LayoutInflater.onCreateView(LayoutInflater.java:938)
	at com.android.internal.policy.PhoneLayoutInflater.onCreateView(PhoneLayoutInflater.java:68)
	at android.view.LayoutInflater.onCreateView(LayoutInflater.java:955)
	at android.view.LayoutInflater.onCreateView(LayoutInflater.java:975)
	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:1029)
	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:986)
	at android.view.LayoutInflater.inflate(LayoutInflater.java:680)
	at android.view.LayoutInflater.inflate(LayoutInflater.java:545)
	at com.forrestguice.suntimes.addon.R$style.showMessage(Unknown Source:17)
	at com.forrestguice.suntimes.intervalmidpoints.MainActivity.onCreate(MainActivity.java:65)
	at android.app.Activity.performCreate(Activity.java:8891)
	at android.app.Activity.performCreate(Activity.java:8856)
	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1468)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3965)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:4136)
	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:103)
	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:149)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:99)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2581)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loopOnce(Looper.java:224)
	at android.os.Looper.loop(Looper.java:318)
	at android.app.ActivityThread.main(ActivityThread.java:8727)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:561)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1013)
Caused by: java.lang.ClassNotFoundException: android.view.x
	... 30 more

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,4 +1,7 @@
 
+### v0.1.3 (2024-11-21)
+* fixes app crash (#9).
+
 ### v0.1.2 (2024-11-18)
 * adds support for system theme (night mode), high contrast themes, and "text size" settings.
 * fixes bug where app icon is not displayed (#5).

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,6 +1,6 @@
 
 ### v0.1.3 (2024-11-21)
-* fixes app crash (#9).
+* fixes app crash in release apk caused by proguard-rules (#9).
 
 ### v0.1.2 (2024-11-18)
 * adds support for system theme (night mode), high contrast themes, and "text size" settings.

--- a/app/build.gradle
+++ b/app/build.gradle
@@ -7,8 +7,8 @@ android {
         applicationId "com.forrestguice.suntimes.intervalmidpoints"
         minSdkVersion 14
         targetSdkVersion 33
-        versionCode 3
-        versionName "0.1.2"
+        versionCode 4
+        versionName "0.1.3"
         buildConfigField "String", "GIT_HASH", "\"${getGitHash()}\""
 
         testInstrumentationRunner 'androidx.test.runner.AndroidJUnitRunner'

--- a/fastlane/metadata/android/en-US/changelogs/4.txt
+++ b/fastlane/metadata/android/en-US/changelogs/4.txt
@@ -1,0 +1,1 @@
+- fixes app crash (#9).

--- a/fastlane/metadata/android/en-US/changelogs/4.txt
+++ b/fastlane/metadata/android/en-US/changelogs/4.txt
@@ -1,1 +1,10 @@
-- fixes app crash (#9).
+v0.1.3
+- fixes app crash in release apk (#9).
+
+v0.1.2
+- adds support for system theme (night mode), high contrast themes, and "text size" settings.
+- fixes bug where app icon is not displayed (#5).
+- fixes bug where battery optimization message is displayed repeatedly (#6).
+- fixes crash when Suntimes is not installed (#7).
+- fixes ANR when Suntimes ContentProvider fails to respond.
+- updates targetSdkVersion (30 -> 33), and migrates app to AndroidX.

